libraries/opensk/src/ctap/client_pin.rs

```diff
@@ -1224,7 +1224,7 @@ mod test {
             ),
             // Reject PIN "12'\0'4" (a zero byte at index 2).
             (
-                b"12\04".to_vec(),
+                b"12\x004".to_vec(),
```

Comment:
nit: `[b'1', b'2', 0, b'4'].to_vec()` might be more readable in this specific case